<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QDialog" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyTimer Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="FirstTimeLabel">
      <property name="font">
       <font>
        <family>Droid Sans [MONO]</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sitting Time (minutes)</string>
      </property>
      <property name="buddy">
       <cstring>FirstTime</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="FirstTime">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="FirstTextLabel">
      <property name="font">
       <font>
        <family>Droid Sans [MONO]</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sitting Text</string>
      </property>
      <property name="buddy">
       <cstring>FirstText</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="FirstText">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="SecondSoundLabel">
      <property name="font">
       <font>
        <family>Droid Sans [MONO]</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sitting Sound File</string>
      </property>
      <property name="buddy">
       <cstring>FirstText</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="StandingBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="StandingFile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="StandingFileBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="StandingPlayBtn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>▶</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="SecondTextLabel">
      <property name="font">
       <font>
        <family>Droid Sans [MONO]</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Standing Time (minutes)</string>
      </property>
      <property name="buddy">
       <cstring>FirstTime</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="SecondTime">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="SecondTimeLabel">
      <property name="font">
       <font>
        <family>Droid Sans [MONO]</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Standing Text</string>
      </property>
      <property name="buddy">
       <cstring>FirstText</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="SecondText">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="FirstSoundLabel">
      <property name="font">
       <font>
        <family>Droid Sans [MONO]</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sitting Sound File</string>
      </property>
      <property name="buddy">
       <cstring>FirstText</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QGroupBox" name="SittingBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="SittingFile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="SittingFileBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SittingPlayBtn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>▶</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FirstTime</tabstop>
  <tabstop>FirstText</tabstop>
  <tabstop>StandingFile</tabstop>
  <tabstop>StandingFileBtn</tabstop>
  <tabstop>SecondTime</tabstop>
  <tabstop>SecondText</tabstop>
  <tabstop>SittingFile</tabstop>
  <tabstop>SittingFileBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Timer</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
